.github/workflows/maven.yml

```diff
@@ -23,8 +23,8 @@ jobs:
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
-        java: [ 11 ]
+        os: [ubuntu-latest, windows-latest]
```

Comment:
I read the title of the PR and was going to suggest to enable the GH Actions for Windows, but glad you've already enabled it! Thanks!